Game/Source/ThirdPersonShooter/Characters/TPSPlayerController.cpp

```diff
@@ -418,6 +418,24 @@ void ATPSPlayerController::DeleteCharacter()
 	}
 }
 
+void ATPSPlayerController::BroadcastStat(const FString& StatCommand)
+{
+	BroadcastStat_ServerWorker(StatCommand);
+}
+
+bool ATPSPlayerController::BroadcastStat_ServerWorker_Validate(const FString& StatCommand)
+{
+	return true;
+}
+
+void ATPSPlayerController::BroadcastStat_ServerWorker_Implementation(const FString& StatCommand)
+{
+	if (ATPSGameState* GS = Cast<ATPSGameState>(GetWorld()->GetGameState()))
```

Comment:
Could the world be nullptr here at any point?

Reply:
it's possible, I'll add a check for that

Reply:
actually I'm unsure, can you have a player controller without a world?

Reply:
I'll add the check anyway, won't hurt